lib/plugin.py

```diff
@@ -34,7 +34,29 @@ def do_get_password_list(self):
             vault_client = Client.create(
                 self.plugin_configuration, self.authentication_username, self.authentication_password
             )
-            return vault_client.get_passwords(self.account, self.asset, self.connection.gateway_username)
+            passwords = vault_client.get_passwords(self.account, self.asset, self.connection.gateway_username)
+            return  {"passwords": passwords}
         except (PluginSDKRuntimeError, RequestException) as ex:
-            self.logger.error("Error retrieving passwords: {}".format(ex))
+            self.logger.error("Error retrieving passwords: %s", ex)
```

Comment:
Is there any chance the we log sensitive data here from the Exception?

Reply:
I don't think so. PluginSDKRuntimeError not throws the original exception and RequestException shouldn't contain any sensitive info